aeron-driver/src/test/java/io/aeron/driver/RetransmitHandlerTest.java

```diff
@@ -77,8 +77,14 @@ class RetransmitHandlerTest
     private final HeaderWriter headerWriter = HeaderWriter.newInstance(
         DataHeaderFlyweight.createDefaultHeader(0, 0, 0));
 
-    private RetransmitHandler handler = new RetransmitHandler(
-        () -> currentTime, invalidPackets, DELAY_GENERATOR, LINGER_GENERATOR, true, retransmitOverflow);
+    private RetransmitHandler handler = new RetransmitHandler(() ->
+        currentTime,
+        invalidPackets,
+        DELAY_GENERATOR,
+        LINGER_GENERATOR,
+        true,
+        Configuration.maxRetransmits(),
```

Comment:
I'd prefer this to be and explicit value in the test as we could end up with non-reproducible tests if system properties started creeping in from somewhere.